kyaml/openapi/openapi.go

```diff
@@ -622,36 +635,46 @@ func parseBuiltinSchema(version string) {
 	assetName := filepath.Join(
 		"kubernetesapi",
 		version,
-		"swagger.json")
+		"swagger.pb")
 
-	if err := parse(kubernetesapi.OpenAPIMustAsset[version](assetName)); err != nil {
-		// this should never happen
-		panic(err)
-	}
-
-	if err := parse(kustomizationapi.MustAsset(kustomizationAPIAssetName)); err != nil {
+	if err := parse(kubernetesapi.OpenAPIMustAsset[version](assetName), "proto"); err != nil {
 		// this should never happen
 		panic(err)
 	}
 }
 
 // parse parses and indexes a single json schema
-func parse(b []byte) error {
+func parse(b []byte, format string) error {
```

Comment:
I'd suggest define `const`s for format string.

Reply:
done, thanks!